microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -714,6 +689,49 @@ WebView2 provides functionality to handle the JavaScript function `window.close(
 
 ---
 
+<!-- ------------------------------ -->
+#### Document title
+
+Your app can detect when the title of the current top-level document has changed.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2.DocumentTitle Property](/dotnet/api/microsoft.web.webview2.core.corewebview2.documenttitle)
+* [CoreWebView2.DocumentTitleChanged Event](/dotnet/api/microsoft.web.webview2.core.corewebview2.documenttitlechanged)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2.DocumentTitle Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#documenttitle)
+* [CoreWebView2.DocumentTitleChanged Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#documenttitlechanged)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2::DocumentTitle property (get)](/microsoft-edge/webview2/reference/win32/icorewebview2#get_documenttitle)<!--no put-->
+* [ICoreWebView2::DocumentTitleChanged event (add](/microsoft-edge/webview2/reference/win32/icorewebview2#add_documenttitlechanged), [remove)](/microsoft-edge/webview2/reference/win32/icorewebview2#remove_documenttitlechanged)
+
+---
+
+<!-- ------------------------------ -->
+#### Favicon 
+
+In WebView2 you can you can set a [Favicon](https://developer.mozilla.org/en-US/docs/Glossary/Favicon) for a website or get notified when it changes. 
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2.FaviconChanged Event](/dotnet/api/microsoft.web.webview2.core.corewebview2.faviconchanged?view=webview2-dotnet-1.0.1293.44&preserve-view=true)
+* [CoreWebView2.FaviconUri Property](/dotnet/api/microsoft.web.webview2.core.corewebview2.faviconuri?view=webview2-dotnet-1.0.1293.44&preserve-view=true)
```

Comment:
delete ?view and &preserve, I think this file nowhere uses those (confirm that).  Maybe need to add hidden comment at top saying "In relnotes .md, include those; in present .md, omit those".

@nishitha-burman  - a note directed to @JasonAndrewWriter and fyi

Reply:
Does this still need to be done?  I am having trouble finding a ?view and &preserve

Reply:
done